multimodal-search-pdf/pods/craft-mminput.yml

```diff
@@ -0,0 +1,16 @@
+jtype: CompoundExecutor
```

Comment:
Rename file to a better name

Reply:
Good catch! This is one I wanted to modify but totally forgot. 😂